x-pack/plugins/security_solution/public/common/utils/validators/index.ts

```diff
@@ -5,16 +5,24 @@
  * 2.0.
  */
 
-import { isEmpty } from 'lodash/fp';
-
 export * from './is_endpoint_host_isolated';
 
-const urlExpression =
-  /(https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]+\.[^\s]{2,}|www\.[a-zA-Z0-9]+\.[^\s]{2,})/gi;
+const allowedSchemes = ['http:', 'https:'];
 
 export const isUrlInvalid = (url: string | null | undefined) => {
-  if (!isEmpty(url) && url != null && url.match(urlExpression) == null) {
-    return true;
+  try {
+    if (url != null) {
```

Comment:
nit: You could maybe combine these as `if (url != null || url === '') {` if you want. Most people do, but this is still fine as is to me as this overall PR improves things.